packages/types/src/type/Extrinsic.ts

```diff
@@ -58,16 +58,9 @@ export default class Extrinsic extends Struct implements IExtrinsic {
           : Compact.addLengthPrefix(u8a)
       );
     } else if (isU8a(value)) {
-      if (!value.length) {
-        return new Uint8Array();
-      }
-
       const [offset, length] = Compact.decodeU8a(value);
-      const total = offset + length.toNumber();
-
-      assert(total <= value.length, `Extrinsic: required length less than remainder, expected at least ${total}, found ${value.length}`);
 
-      return value.subarray(offset, total);
+      return value.subarray(offset, offset + length.toNumber());
```

Comment:
I really don't like the fact that we dropped the assert here.

Reply:
Sorry, copy-pasted and forgot to check